File: test/java/arraysandstrings/SquaresOfASortedArrayTest.java
package java.arraysandstrings;

import org.junit.jupiter.api.Test;

import static java.arraysandstrings.SquaresOfASortedArray.sortedSquares;
import static org.junit.jupiter.api.Assertions.assertArrayEquals;

class SquaresOfASortedArrayTest {

    @Test
    void sortedSquaresTest() {
        assertArrayEquals(new int[]{4,9,9,49,121},
                sortedSquares(new int[] {-7,-3,2,3,11}),
                "Sorted squares test");
    }
}

File: src/test/kotlin/arraysandstrings/SquaresOfASortedArrayTest.kt
package arraysandstrings

import org.junit.jupiter.api.Assertions.assertArrayEquals
import org.junit.jupiter.api.Test

class SquaresOfASortedArrayTest {
    @Test
    fun sortedSquaresTest() {
        assertArrayEquals(intArrayOf(4, 9, 9, 49, 121),
            SquaresOfASortedArray().sortedSquares(intArrayOf(-7, -3, 2, 3, 11)))
    }

}